core/auth/local/user_management.py

```diff
@@ -20,7 +20,8 @@
     "oneshotanalytics": {"read": True, "write": True, "toggle": True, "run": True},
     "tag": {"read": True, "write": True},
     "export": {"read": True, "write": True, "toggle": True, "refresh": True},
-    "files": {"read": True, "write": True},
+    "attachedfiles": {"read": True, "write": True},
+    "file": {"read": True, "write": True},
```

Comment:
We need to find a better way to do this.

If we merge this, users will need to edit the database manually in order to add the permissions.

Maybe the use of a script in order to update Yeti that would do the git pull and apply some sort of basic migrations ? A "migration" could simply be a script (which could even be interactive). This would imply some sort of status for each "migration" in the database (applied / error). Any other idea / option to do this properly ?